src/components/sections/functions.js

```diff
@@ -1,23 +1,9 @@
 import React from "react";
 import { Typography, Grid } from "@mui/material";
-import { MaxWidthWrapper } from "../layout/maxWidthWrapper";
 
-export const FunctionsSection = ({ content }) => {
-  return (
-    <MaxWidthWrapper gutterBottom>
-      <Typography variant="h2">
-        What can HeLx do for you?
-      </Typography>
-      <Grid container spacing={{ md: 8, sm: 6, xs: 4 }}>
-        {content.map((feature, index) => (
-          <Grid item md={6} xs={12} key={index}>
-            <Typography variant="h3">
-              {feature.heading}
-            </Typography>
-            <Typography variant="body1">{feature.paragraph}</Typography>
-          </Grid>
-        ))}
-      </Grid>
-    </MaxWidthWrapper>
-  );
-};
+export const FunctionsGridItem = ({ heading, children }) => (
+  <Grid item md={6} xs={12}>
+    <Typography variant="h3">{heading}</Typography>
```

Comment:
remove `<Typography/>` components. you shouldnt need to wrap `{children}` in a Typography component b/c the markdown mapping should take care of that. for this component, see my note in `src/content/features/helxFunctions.mdx` about removing the heading prop and just passing it as a `### heading3` in the markdown